Configurations/10-main.conf

```diff
@@ -1717,6 +1717,7 @@ my %targets = (
         CFLAGS           => picker(default => "-Wall",
                                    debug   => "-g -O0",
                                    release => "-O3"),
+        ex_libs          => add("-lcrypt32"),
```

Comment:
Noting that the 64bit version of this library is still called `crypt32.lib`, but in a different architecture directory.

Reply:
Goodness, this. Accordingly we should successfully be able to test "win32" and "win64". See in https://github.com/openssl/openssl/commit/f09f774fbb35f03866f0e463a815032c1b7d7375 .

Reply:
It was really just a note to me, to not confuse 32-bit vs 64-bit... sigh.